procedures/automation-android-setup.md

```diff
@@ -0,0 +1,200 @@
+        * Add `$pin_puppet_server = 'releng-puppet2.srv.releng.mdc1.mozilla.com'`
+        * Add `$pin_puppet_env = '$username'`
+    10. SSH into both the pushapk worker (`dep-m-pushapkworker-1.srv.releng.use1.mozilla.com`) and the signing worker (`dep-m-signing-linux-1.srv.releng.use1.mozilla.com`)
+        * Run `sudo puppet agent --test --server=releng-puppet2.srv.releng.mdc1.mozilla.com --environment=$username` (may take a long time)
+        * Run `sudo puppet agent --test` again
+    11. In [Hooks](https://taskcluster-web.netlify.com/hooks), go to the staging hook you created earlier
+    12. Run a staging build
+        1. Click "Trigger Hook" at the bottom of the page
+        2. Under "Last Fired Result" (near the top), click the link
+        3. Once the build is finished, go to the log, scroll to the bottom and copy the generated ID (to the right of the "RESULT" text)
+        4. Go to [Task Groups](https://taskcluster-web.netlify.com/tasks/groups) and search with the value you copied
+    13. Once everything is working (up to `pushapk` failing due to the certificate being empty), that means that all automation is working!
+13. Once step 11's PR is approved and the build is working in step 12, merge the `build-puppet` PR
+14. Verify with app's team how `versionCode` should be set up. Perhaps by date like [`fenix`](https://github.com/mozilla-mobile/fenix/blob/master/automation/gradle/versionCode.gradle)?
+    * Note that if there's multiple build types, they need different version codes. In the case of [`fenix`](https://github.com/mozilla-mobile/fenix/blob/master/app/build.gradle#L50-L52), `x86` builds have the version code incremented by 1.
+15. When the Google Play product is being set up, an officially-signed build with a version code of 1 needs to be built. So, the main automation PR for the product will need to be stunted: it needs to produce APKs with a version code of 1, and it should have pushing to Google Play disabled (so we don't accidentally push a build before our official version-code-1 build is set up).
+    1. Change the version code to be set to 1. If the product uses the same version-code-by-date schema as `fenix`, then edit [`versionCode.gradle`](https://github.com/mozilla-mobile/fenix/pull/156/files#diff-63606bb315fadc051f73a54767849985R41)
+    2. [Disable the creation of the task that pushes to Google Play](https://github.com/mozilla-mobile/fenix/pull/156/files#diff-73e51d972c105de5122ec559909980daR123)
+    3. Create the PR
+    4. Once approved, merge the PR
+16. Verify the apk artifact(s) of the signing task
+    1. Trigger the non-staging hook
+    2. Once the build finishes, download the apks from the signing task
+    3. Using the prod certificate from step 10.4.1., create a temporary keystore: `keytool -import -noprompt -keystore tmp_keystore -storepass 12345678 -file $product_release.pem -alias $product-rel`
```

Comment:
Nit: Github renders the numbered list as 10.iv.a (instead of 10.4.1). I'm unsure on how to properly fix this one. It depends on the configuration of the markdown parser

Reply:
I changed the text to `10.iv.a` ez :smile: